loopd/swapclient_server.go

```diff
@@ -508,6 +512,59 @@ func (s *swapClientServer) SwapInfo(_ context.Context,
 	return s.marshallSwap(&swp)
 }
 
+// AbandonSwap requests the server to abandon a swap with the given hash.
+func (s *swapClientServer) AbandonSwap(_ context.Context,
```

Comment:
Here we are not actually making use of the request context, but the current implementation of `AbandonSwap` is writing to a channel that is not buffered. Why not pass down the request context to the methods?